core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -316,14 +316,22 @@ abstract class SparkFunSuite
     } else {
       assert(expectedParameters === parameters)
     }
-    val actualQueryContext = exception.getQueryContext()
-    assert(actualQueryContext.length === queryContext.length, "Invalid length of the query context")
-    actualQueryContext.zip(queryContext).foreach { case (actual, expected) =>
-      assert(actual.objectType() === expected.objectType(), "Invalid objectType of a query context")
-      assert(actual.objectName() === expected.objectName(), "Invalid objectName of a query context")
-      assert(actual.startIndex() === expected.startIndex(), "Invalid startIndex of a query context")
-      assert(actual.stopIndex() === expected.stopIndex(), "Invalid stopIndex of a query context")
-      assert(actual.fragment() === expected.fragment(), "Invalid fragment of a query context")
+    if (!queryContext.isEmpty) {
```

Comment:
Please, remove this check. Test must reflect current imple of exception. If exceptions have already had query contexts, test should check them.

Reply:
Hmm, there was a reason for this.. I'll refresh my memory.

Reply:
OK, I have removed all that and checked the contexts (very painful!) and of dubious value IMHO.